examples/basic/add/run.py

```diff
@@ -4,11 +4,15 @@
 from oremda import Operator as OremdaOperator
 
 class AddOperator(OremdaOperator):
-    def kernel(self, input_data, parameters):
+    def kernel(self, meta, data, parameters):
```

Comment:
So if there are multiple input data arguments, how do we make sure we pass the right array to the right data object? Is the name for each argument being supplied in the `pipeline.json`, so they get matched correctly?

Reply:
Yes, it's the pipeline responsibility to define connections between the named ports on the operators and sources

Reply:
Sounds good, thanks!